### DISCUSSION TITLE
Highway/Path in Openstreetmap converts to Highway/Via_Ferrata in Osmand

### DISCUSSION THREAD

**Discussion by christofkn** [osmandapp/OsmAnd#12823] [Maps Related]
Some Paths that are tagged as Highway/Path in Openstreetmap are recently tagged as Highway/Via_Ferrata in Osmand maps.
I can not access them via tag="highway" value="path" but only via tag="highway" value="via_ferrata" in my render.xml.
Same happens when I create a map with Osmand MapCreator.

This seems to happen everythime, when a path has both tags, "sac_scale" and "via_ferrata_scale" in Openstreetmap.

The problem is, that there are many paths that are very difficult to go, but have few, easier via-ferrata sections (e.g. Jubiläumsgrat).
In Openstreetmap they are (correctly) tagged as highway-path (not highway-via_ferrata) with high "sac_scale" and low "via_ferrata_scale".
In Osmand they appear as via_ferrata with low via_ferrata_scale, which is wrong. 
I have no chance to change this behavior in my own renderer, since this error happens in map creation process.

In older maps (I think, some months ago) everything was fine.

Can you perhaps change map creation process back to the state before, when highway/path with sac_scale and via_ferrata_scale in Openstreetmap were still tag="highway" value="path" in Osmand?

**Comment by xmd5a2** [comment]
> In Osmand they appear as via_ferrata with low via_ferrata_scale, which is wrong.

Can you explain this in more detail?

**Reply by christofkn** [reply]
Solved the issue (for me).
In Osmand MapCreator/rendering_types.xml I deleted the line 
<entity_convert pattern="tag_transform" from_tag="via_ferrata_scale" to_tag1="via_ferrata_scale" to_tag2="highway" to_value2="via_ferrata"/>
and everything is fine now in the maps I create.

One thing to consider:
With this every line with additional tag "via_ferrata_scale" is transformed to highway/via_ferrata in Osmand.  
This helps, when via ferratas are tagged in old style in Openstreetmaps (highway/path instead of highway/via_ferrata).
On the other hand it creates mistakes, when a path is deliberately tagged as highway/path and for some reason has an additional tag "via_ferrata_scale".

Example Jubiläumsgrat:
In Openstreetmaps it is highway/path with additional tags sac_scale=6 and via_ferrata_scale=4.
In reality it is an extremely difficult path with (few) cables at some locations. The cable sections are still demanding (via_ferrata_scale=4), but they are the easier part of the path. Much more difficult are the sections without any safety device (sac_scale=6).
In Osmand it appears as highway/via_ferrata with via_ferrata_scale=4. But it should appear as highway/path with sac_scale=6.

It is striking, that there appear many via ferratas in Osmand, that are no via ferratas in reality.

**Reply by vshcherb** [reply]
Can you please create a pull request to check it.

**Comment by xmd5a2** [comment]
Ok, I fixed that https://github.com/osmandapp/OsmAnd-resources/commit/1e7d033dea29dee6e30146e920503ede41953fb2

**Comment by drazde** [comment]
Hi,
is it possible render every 
highway=path
via_ferrata=*

like a via ferrata?
Yesterday I hike on dolomites and in this stretch there is a little easy ferrata but it is not displayed

![image](https://user-images.githubusercontent.com/2643448/177199040-43fa87bf-d19a-4128-ae5f-bb617d130643.png)
